src/models/StyleSheet.js

```diff
@@ -5,4 +5,13 @@
 
 import { StyleSheet } from '../vendor/glamor/sheet'
 
-export default new StyleSheet({ speedy: false, maxLength: 40 })
+class StyleSheetExtended extends StyleSheet {
+  reset() {
+    return super.flush()
+  }
+  getCSS() {
+    return super.rules().map(rule => rule.cssText).join('\n')
```

Comment:
Why are we including linebreaks by default? Shouldn't we just do `.join('')` for performance reasons, so that the code the developers send down to the client is as small as possible?


If we want to allow devs to get non-minified CSS (for debugging purposes),  maybe we could have an option called `min` which is set to true by default?

```JS
getCSS({ min = true }) {
  return super.rules().map(rule => rule.cssText).join(min ? '' : '\n')
}
```

That way devs can do normal mode, which is minified, or non-minified mode if they want to debug:

```
// minified
styleSheet.getCSS()

// Not minified
styleSheet.getCSS({ min: false })
```